notifications/base/models.py

```diff
@@ -237,9 +237,9 @@ class Meta:
         abstract = True
         ordering = ('-timestamp',)
         # speed up notifications count query
-        index_together = ('recipient', 'unread')
         verbose_name = _('Notification')
         verbose_name_plural = _('Notifications')
+        indexes = [models.Index(fields=["recipient", "unread"])]
```

Comment:
are we sure this is backward compatible with older django versions?

Reply:
FYI - it has been there [since Django 1.1.1](https://docs.djangoproject.com/en/1.11/ref/models/indexes/) at least